docs/_basic/web_api.md

```diff
@@ -6,21 +6,21 @@ order: 4
 ---
 
 <div class="section-content">
-You can call [any Web API method](https://api.slack.com/methods) using the [`WebClient`](https://slack.dev/node-slack-sdk/web-api) provided to your Bolt app as `app.client` (given that your app has the appropriate scopes). When you call one the client's methods, it returns a Promise containing the response from Slack.
+You can call [any Web API method](https://api.slack.com/methods) using the [`WebClient`](https://slack.dev/node-slack-sdk/web-api) provided to your app's listeners as `client`. This uses either the token that initialized your app <b>or</b> the token that is returned from the [`authorize` function](#authorization) for the incoming event. The built-in [OAuth support](#authenticating-oauth) handles the second case by default.
```

Comment:
This is correct, but I do wonder if it will lead to confusion as people go to the authorize section. Maybe we should add a note to the authorize section saying it isn't needed. if using the build-in. OAuth support.